presentation/src/main/java/org/sagebionetworks/research/presentation/show_step/show_step_view_models/ShowUIStepViewModel.java

```diff
@@ -88,6 +88,15 @@ public void handleAction(final String actionType) {
             case ActionType.BACKWARD:
                 performTaskViewModel.goBack();
                 break;
+            case ActionType.SKIP:
+                performTaskViewModel.goForward();
+                break;
+            case ActionType.CANCEL:
```

Comment:
For the ones we haven't implemented, we should at least log a warning. Same with the default case. 

So let's log a warning instead of an UnsupportedOperationException and have CANCEL, INFO, and ADD_MORE fall through to the `default` case

Reply:
Let's log at a higher level -- `LOGGER.warn`. 

When these lines are reached, it means the dev have done something wrong which the app is recovering from (in this case by doing ignoring the action). 

Previously, we were crashing to make it even more obvious the devs have done something wrong (harder for dev and/or tester to gloss over a bug this way)